style.css

```diff
@@ -29,33 +28,31 @@ header {
 .home {
   display: flex;
   flex-direction: column;
-  width: 375px;
 }
 
 .bars {
   display: flex;
   align-items: center;
-  width: 375px;
-  justify-content: space-around;
+  justify-content: space-between;
   background: white;
+  padding: 0 3vw;
 }
 
 .toolbar {
   display: flex;
   align-items: center;
-  justify-content: space-around;
-  width: 100%;
+  justify-content: space-between;
+  padding: 0 5vw;
+  width: 90vw;
 }
 
 .left {
-  width: 45%;
   right: 12px;
```

Comment:
The "right" property could only be used when the "display : absolute" property is included.
It may be a better way to exclude it or add the "display: absolute" to effect the "right" property